### PR TITLE
Add warnings for bin/cluster and READMEs

### PR DESCRIPTION
**Pull request by abutcher**
How do you guys feel about a message like this for `bin/cluster`? @brenton @detiber @sdodson @adellape

### CODE REVIEW

**Comment by detiber** [comment]
:heart_eyes:

File: README_AWS.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+:warning: **WARNING** :warning: This feature is community supported and may contain bugs. Visit [docs.openshift.com](https://docs.openshift.com) for [OpenShift Enterprise](https://docs.openshift.com/enterprise/latest/install_config/install/index.html) or [OpenShift Origin](https://docs.openshift.org/latest/install_config/install/index.html) supported installation docs.
```
Comment (sdodson) [inline README_AWS.md]: Swap `may contain bugs` with `has not been tested by Red Hat` ? Just trying to avoid the implication that other bits are bug free.

**Comment by brenton** [comment]
This makes sense to me.  Let me know when you have the wording to your liking.

**Comment by abutcher** [comment]
I'll give the wordsmith @adellape a chance to chime in.

**Comment by adellape** [comment]
@abutcher Straight :fire:

**Comment by abutcher** [comment]
@brenton Burn it down

**Comment by detiber** [comment]
image

**Pull request merged**